examples/http-client-data-binding/http_client_data_binding.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - Payload data binding
 
-HTTP client payload data binding feature allows accessing the response payload directly without manipulating the `http:Response` object. The response payload type is inferred from the contextually expected type or the `targetType` argument. `anydata` type or `http:Response` is expected as the expected-type or the `targetType` along with `error` type. When the client payload data binding is expected, the HTTP error responses (4xx, 5xx) are categorized as an `error` (`http:ClientRequestError`, `http:RemoteServerError`) of the client's remote operation. If the expected payload type is not compatible with the response payload type an `http:ClientError` is returned. Use this to access the response payload directly when the response payload type is known beforehand. 
+The `http:Client` payload data-binding allows directly binding the response payload to a given subtype of `anydata`. It does this by mapping a given HTTP content-type to one or more Ballerina types. For instance, `text/plain` is mapped to `string` whereas `application/json` is mapped to `json`, `record`, etc. Client data-binding can be used by simply assigning the resource method’s returned value to the declared variable. If the response is anything other than 2xx, an `error` is returned and no data-binding is performed. If there is no mapping between the given Ballerina type and the response content-type, again an `error` is returned. Use this when the application is only interested in the response payload but not the headers. When the response payload is JSON, `record` type is preferred to `json` type as it provides compile-time validations, better readability and improved tooling support.
```

Comment:
```suggestion
The `http:Client` payload data binding allows binding the response payload directly to a given subtype of `anydata`. It does this by mapping a given HTTP content-type to one or more Ballerina types. For instance, `text/plain` is mapped to `string` whereas `application/json` is mapped to `json`, `record`, etc. 

Client data binding can be used by simply assigning the resource method’s returned value to the declared variable. If the response is anything other than 2xx, an `error` is returned and no data binding is performed. If there is no mapping between the given Ballerina type and the response content-type, again an `error` is returned. Use this when the application is only interested in the response payload but not the headers. When the response payload is JSON, the `json` type is preferred as the `record` type as it provides compile-time validations, better readability, and improved tooling support.
```
This paragraph is too lengthy. Shall we break it into 2? Also, check if the technical meaning is affected by the suggested change on the `json` type and the `record type`. @ayeshLK

Reply:
It is only 140 words. So one paragraph should be okay, IMO. Remember, when you render it in the website it looks a lot more smaller.